packages/zowe-explorer/src/PersistentFilters.ts

```diff
@@ -254,29 +254,13 @@ export class PersistentFilters {
     private initialize(): void {
         ZoweLogger.trace("PersistentFilters.initialize called.");
         const settings = ZoweLocalStorage.getValue<PersistentFilter>(this.schema);
-        let searchHistoryLines: string[];
-        let sessionLines: string[];
-        let fileHistoryLines: string[];
-
         if (settings) {
-            searchHistoryLines = settings[PersistentFilters.searchHistory];
-            sessionLines = settings[PersistentFilters.sessions];
-            fileHistoryLines = settings[PersistentFilters.fileHistory];
-        }
-        if (searchHistoryLines) {
-            this.mSearchHistory = searchHistoryLines;
-        } else {
-            this.resetSearchHistory();
-        }
-        if (sessionLines) {
-            this.mSessions = sessionLines;
-        } else {
-            this.resetSessions();
-        }
-        if (fileHistoryLines) {
-            this.mFileHistory = fileHistoryLines;
-        } else {
-            this.resetFileHistory();
+            this.mSearchHistory = settings[PersistentFilters.searchHistory] ?? [];
+            this.mSessions = settings[PersistentFilters.searchHistory] ?? [];
+            this.mFileHistory = settings[PersistentFilters.searchHistory] ?? [];
```

Comment:
Looks like we are assigning the search history to all 3 arrays. Should it be updated to something like this?

```ts
this.mSessions = settings[PersistentFilters.sessions] ?? [];
this.mFileHistory = settings[PersistentFilters.fileHistory] ?? [];
```